### ISSUE TITLE
Multiple Layout Cops throwing false errors for indentation and alignment checks

### ISSUE THREAD

**Issue by daytonpa** [chef/cookstyle#358]
# Version:
```
$ chef exec cookstyle --version
Cookstyle 5.10.13
  * RuboCop 0.72.0
```

# Environment:
macOS Mojave (10.14.6)

Running Cookstyle checks on managed InSpec Profile
```shell
$ chef exec cookstyle -D .
```

# Scenario:

Cookstyle is throwing tons of errors regarding indentation and alignment.  Looking at my ruby files, indentation is fine and consistent across the board, but I am encountering 100+ errors from the syntax checker.

The included Layout cops throwing errors are:
- Layout/IndentationConsistency
- Layout/BlockAlignment
- Layout/IndentationWidth
- Layout/AlignArray

# Steps to Reproduce:

Unfortunately, I cannot post exact code snippets due to company property.  However, I will provide examples of each error I am encountering with slightly modified values

### Example 1: IndentationConsistency
```
controls/example.rb:427:3: C: Layout/IndentationConsistency: Inconsistent indentation detected.
  describe file('/etc/ssh/sshd_config') do ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

```ruby
control 'test-11' do
  impact 1.0
  title 'os config'
  desc 'check for sshd configurations'

  describe file('/etc/ssh/sshd_config') do
    it { should exist }
    its('owner') { should eq 'root' }
    its('group') { should eq 'root' }
  end
end
```
### Example 2: BlockAlignment
```
controls/example.rb:432:2: C: Layout/BlockAlignment: end at 432, 1 is not aligned with describe file('/etc/ssh/sshd_config') do at 427, 2.
	end
 ^^^
```

```ruby
control 'test-11' do
  impact 1.0
  title 'os config'
  desc 'check for sshd configurations'

  describe file('/etc/ssh/sshd_config') do
    it { should exist }
    its('owner') { should eq 'root' }
    its('group') { should eq 'root' }
  end
end
```

### Example 3: IndentationWidth
```
controls/example.rb:656:1: C: Layout/IndentationWidth: Use 2 (not 1) spaces for indentation.
	impact 1.0
^
```

```ruby
control 'test-17' do
  impact 1.0
  title 'files'
  desc 'check for some files'
  
  %w(
    /path/to/a/file
    /path/to/b/file
    /path/to/c/file
    /path/to/d/file
    /path/to/e/file
    /path/to/f/file
  ).each do |file_path|
    describe file(file_path) do
      it { should exist }
    end
  end
end
```
### Example 4: AlignArray
```
controls/example.rb:663:3: C: Layout/AlignArray: Align the elements of an array literal if they span more than one line.
		/path/to/e/file
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

```ruby
control 'test-17' do
  impact 1.0
  title 'files'
  desc 'check for some files'
  
  %w(
    /path/to/a/file
    /path/to/b/file
    /path/to/c/file
    /path/to/d/file
    /path/to/e/file
    /path/to/f/file
  ).each do |file_path|
    describe file(file_path) do
      it { should exist }
    end
  end
end
```

# Expected Result:

No alignment errors should be thrown.  Everything is lined up properly within my editor and should not be causing any issues with the syntax checker for the listed Layout Cops.

# Actual Result:
```3 files inspected, 102 offenses detected```

**Comment by tas50** [comment]
@daytonpa Any chance you could send me the entire profile so I could track this down. I'm having a hard time reproducing it with just control snippets. You can e-mail me at tsmith@chef.io

**Comment by tas50** [comment]
@daytonpa I've updated our copy of Rubocop from 0.72 to 0.75, which includes a large number of bug fixes. This may resolve these (and other) problems. The indentation code actually lives in built-in Rubocop cops. If these aren't resolved with 0.75 I'll get an upstream issue created and linked to this.

**Comment by tas50** [comment]
@daytonpa I tried to reproduce this on 5.10 and 5.11 with the snippets you provided and neither versions alert for any formatting issues. Do you have anything set in a .rubocop.yml that might be causing those errors or can you give a more complex and sanitized version of the original control?

**Comment by tyler-ball** [comment]
We have released a few more versions of cookstyle since this was reported and we cannot reproduce the error. @daytonpa please re-open this if you are still seeing errors using the latest version of Chef Workstation!
